src/main/scala/fr/polytechnique/cmap/cnam/etl/extractors/acts/SsrCeActExtractor.scala

```diff
@@ -0,0 +1,45 @@
+package fr.polytechnique.cmap.cnam.etl.extractors.acts
+
+import java.sql.{Date, Timestamp}
+import org.apache.spark.sql.{DataFrame, Row, functions}
+import fr.polytechnique.cmap.cnam.etl.events.{Event, SsrCEAct, MedicalAct}
+import fr.polytechnique.cmap.cnam.etl.extractors.Extractor
+import fr.polytechnique.cmap.cnam.etl.sources.Sources
+import fr.polytechnique.cmap.cnam.util.datetime.implicits._
+
+object SsrCeActExtractor extends Extractor[MedicalAct] with SsrCeSourceExtractor {
+  override def isInStudy(codes: Set[String])
+    (row: Row): Boolean = codes.exists(getCode(row).startsWith(_))
+
+  override def isInExtractorScope(row: Row): Boolean = !isNullAt(ColNames.CamCode)(row)
+
+  override def builder(row: Row): Seq[Event[MedicalAct]] = {
+    lazy val patientID = getPatientID(row)
+    lazy val date = getDate(row)
+    lazy val code = getCode(row)
+    // todo + tard, on peut recuperer le ETA_NUM + numero du pfs de santé à la place de ACE pr groupID
```

Comment:
We need to discuss this subject. I imagine that we must agree on the role of a GroupId and stick to it for all the Extractors. This could be the object of an other PR in my opinion.